### ISSUE TITLE
can not import string keys csv

### ISSUE THREAD

**Issue by zhyhang** [pilosa/pilosa#1575]
## import string keys csv file, rows and columns are not real imported
- version 1.0.2
- reproduce steps:

```bash
./pilosa server
curl localhost:10101/index/test1 -d '{"options": {"keys": true}}' -XPOST
curl localhost:10101/index/test1/field/f1 -d '{"options": {"keys": true}}' -XPOST
./pilosa import -i test1 -f f1 --string-keys import.test.csv

# get the message:
2018/08/02 22:11:12 parsing: import.test.csv
2018/08/02 22:11:12 importing keys: n=2

curl localhost:10101/index/test1/f1 -d 'Row(daat="row2")'
# output:
# {"results":[{"attrs":{},"columns":[]}]}

cat import.test.csv
# output:
row1,col1
row2,col2

```

**Comment by jaffee** [comment]
This now works, with the caveat that there is a bug with key imports that needs to be solved #1577 

We'll leave this open until that's fixed and verified.

**Comment by zhyhang** [comment]
Master has merged the import keys!
